app/models/shoe.rb

```diff
@@ -0,0 +1,10 @@
+class Shoe < ApplicationRecord
+  validates :model, presence: true
```

Comment:
I had to check that `model` wasn't a Rails reserved word. But, to my surprise, it's not! So you're all good.